src/top-bar/period-select/period-select.js

```diff
@@ -45,15 +50,20 @@ const PeriodSelect = () => {
             onClose={() => setOpenedSelect('')}
             requiredValuesMessage={i18n.t('Choose a workflow first')}
         >
-            <YearNavigator
-                maxYear={maxYear}
-                year={year}
-                onYearChange={year => {
-                    selectPeriod(null)
-                    setYear(year)
-                }}
-            />
-            <PeriodMenu periodType={workflow?.periodType} year={year} />
+            {year && (
+                <>
+                    <YearNavigator
+                        maxYear={maxYear}
+                        year={year}
+                        onYearChange={year => {
+                            selectPeriod(null)
+                            setYear(year)
+                        }}
+                    />
+
+                    <PeriodMenu periodType={workflow?.periodType} year={year} />
+                </>
+            )}
```

Comment:
This part is more of a safeguard. It should only be relevant if the `workflow` doesn't have a `periodType` and I believe this is a serious configuration error. I think this is more of a hypothetical issue. If it turns out to actually happen once the app is deployed, we might want to show some sort of warning if there is no `year`.